### PR TITLE
Style mostly there, about page and resume page + navigation all up!

### PR DESCRIPTION
**Pull request by AlvieH**

### CODE REVIEW

**Review by zkoogle — COMMENTED** [review]
Looks like you migrated a library of a sort? If you did library migration and your own code change, can you please separate them?

**Comment by AlvieH** [comment]
Removed those files, thanks!

File: portfolio/src/main/webapp/style.css [hunk 1]
```diff
@@ -1,9 +1,189 @@
...
+    width: 100%;
+    max-width: 1000px;
+    height: 550px;
+    margin: 2em 0;
+    display: inline-block;
+}
+
+.pdfobject {
+    border: solid 3px #ff5555;
+}
```
Comment (askewc) [inline portfolio/src/main/webapp/style.css]: new line

File: portfolio/src/main/webapp/style.css [hunk 2]
```diff
@@ -1,9 +1,189 @@
...
+
+.pdfobject-container {
+    width: 100%;
+    max-width: 1000px;
+    height: 550px;
+    margin: 2em 0;
+    display: inline-block;
+}
+
+.pdfobject {
```
Comment (askewc) [inline portfolio/src/main/webapp/style.css]: pdf-object & pdf-object-container? Or is this part of a library?

File: portfolio/src/main/webapp/style.css [hunk 3]
```diff
@@ -1,9 +1,189 @@
+@font-face {
```
Comment (askewc) [inline portfolio/src/main/webapp/style.css]: tab width 2 spaces

File: portfolio/src/main/webapp/resume.html [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    <ul class="contact-links">
+    </ul>
+  </nav>
+</header>
+<body>
+    <div class="div-wrapper">
+      <div id="resume"></div>
+    </div>
+    <script>PDFObject.embed("/files/AlvinHermansResume.pdf", "#resume");</script>
+</body>
```
Comment (askewc) [inline portfolio/src/main/webapp/resume.html]: newline

File: portfolio/src/main/webapp/resume.html [hunk 5]
```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<head>
+  <meta charset="UTF-8">
+  <title>Alvin Hermans</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto+Mono&display=swap">
+  <script src="script.js"></script>
```
Comment (askewc) [inline portfolio/src/main/webapp/resume.html]: can you move these scripts to the bottom of `<body>`

File: portfolio/src/main/webapp/script.js [hunk 6]
```diff
@@ -26,3 +26,12 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+/**
+* Plays a specified melody
+ */
+function play(melody) {
+    path = "files/audio/";
```
Comment (askewc) [inline portfolio/src/main/webapp/script.js]: Does `path` get defined elsewhere? If not, plz use const to declare. Also, I this isn't the `path`. Maybe move it to the top of the file and name it `AUDIO_DIR`?

File: portfolio/src/main/webapp/script.js [hunk 7]
```diff
@@ -26,3 +26,12 @@ function addRandomGreeting() {
...
+    var sound = new Audio(path + melody + ".mp3");
```
Comment (askewc) [inline portfolio/src/main/webapp/script.js]: prefer `const` over `var`.

File: portfolio/src/main/webapp/script.js [hunk 8]
```diff
@@ -26,3 +26,12 @@ function addRandomGreeting() {
...
+    sound.play();
+}
```
Comment (askewc) [inline portfolio/src/main/webapp/script.js:147]: need a newline

File: portfolio/src/main/webapp/index.html [hunk 9]
```diff
@@ -1,18 +1,24 @@
...
+          <li><a href="about.html">About Me</a></li>
+          <li><a href="resume.html">Resume</a></li>
+        </ul>
+        <ul class="contact-links">
+        </ul>
+      </nav>
+    </header>
+    <body>
+    </body>
+</html>
```
Comment (askewc) [inline portfolio/src/main/webapp/index.html]: newline

**Review by askewc — CHANGES_REQUESTED** [review]
package-lock! are you using nodejs?

Overall, this looks solid! Just some style things.

File: portfolio/src/main/webapp/style.css (hunk 2 above)
Comment (AlvieH) [reply portfolio/src/main/webapp/style.css]: Part of a library - I think the container is the div that's holding the pdf and the object is the pdf itself?

File: portfolio/src/main/webapp/style.css (hunk 1 above)
Comment (AlvieH) [reply portfolio/src/main/webapp/style.css]: done!

File: portfolio/src/main/webapp/resume.html (hunk 5 above)
Comment (AlvieH) [reply portfolio/src/main/webapp/resume.html]: deleted instead, they weren't used on that page anyway but I can add back later if needed

File: portfolio/src/main/webapp/script.js (hunk 6 above)
Comment (AlvieH) [reply portfolio/src/main/webapp/script.js]: done!

File: portfolio/src/main/webapp/script.js (hunk 7 above)
Comment (AlvieH) [reply portfolio/src/main/webapp/script.js]: done!

**Comment by AlvieH** [comment]
fixed the style stuff, as well as the icons that weren't rendering correctly on the navbar. Main navbar fonts seemed to break though? idk why

File: portfolio/src/main/webapp/script.js (hunk 8 above)
Comment (AlvieH) [reply portfolio/src/main/webapp/script.js:147]: done!

**Review by AlvieH — COMMENTED** [review]
Looks good! Incorporated these changes into the last commit :)

**Pull request merged**